GitUI/HelperDialogs/FormRunScriptSpecify.Designer.cs

```diff
@@ -39,24 +39,34 @@ private void InitializeComponent()
             this.specifyLabel.AutoSize = true;
             this.specifyLabel.Location = new System.Drawing.Point(35, 13);
             this.specifyLabel.Name = "specifyLabel";
-            this.specifyLabel.Size = new System.Drawing.Size(35, 13);
+            this.specifyLabel.Size = new System.Drawing.Size(38, 15);
             this.specifyLabel.TabIndex = 0;
             this.specifyLabel.Text = "label1";
             // 
             // branchesListView
             // 
+            this.branchesListView.Anchor = ((System.Windows.Forms.AnchorStyles)((((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Bottom) 
+            | System.Windows.Forms.AnchorStyles.Left) 
+            | System.Windows.Forms.AnchorStyles.Right)));
+            this.branchesListView.Columns.AddRange(new System.Windows.Forms.ColumnHeader[] {
+            this.dummyColumnHeader});
             this.branchesListView.FullRowSelect = true;
             this.branchesListView.HeaderStyle = System.Windows.Forms.ColumnHeaderStyle.None;
             this.branchesListView.Location = new System.Drawing.Point(12, 40);
             this.branchesListView.Name = "branchesListView";
             this.branchesListView.Size = new System.Drawing.Size(290, 52);
             this.branchesListView.TabIndex = 1;
             this.branchesListView.UseCompatibleStateImageBehavior = false;
-            this.branchesListView.View = System.Windows.Forms.View.List;
+            this.branchesListView.View = System.Windows.Forms.View.Details;
+            // 
+            // dummyColumnHeader
+            // 
+            this.dummyColumnHeader.Text = "<dummy column>";
```

Comment:
The translation app may pick up this.

Reply:
I guess I should leave the column text empty. It is not visible anyway and the text is not used for sizing, either.